### ISSUE TITLE
Generate WCAG compliant Images for Backgrounds instead of using files in `img_bg

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/LGBTQ-of-the-day-bot#265]
**Is your feature request related to a problem? Please describe.**
Currently, a light-colored translucent box is used on top of the background images. It was used as a way to make the words accessible, but it's hindering the legibility in a lot of cases.

**Describe the solution you'd like**
Generate a background image that complies with WCAG a11y standards.

**Describe alternatives you've considered**
randomized pride flag colored backgrounds masked to lighten them up and allow for AA accessibility

**Additional context**
Currently using images in the `img_bg` folder. `get_img_for_tweet.py` is used to add text to those images.
#2 
#250 
#147 
https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.new
https://alexwlchan.net/2020/03/stripey-flag-wallpapers/
https://alexwlchan.net/2016/10/wallpapers-with-pillow/
https://alexwlchan.net/2016/10/tiling-the-plane-with-pillow/

---
### Image Requirements
- in one of the standard fonts, include twitter handle `@LGBTQotd` somewhere on the image
- **Must** adhere to WCAG accessibility standards for contrast and color blindness with `#0b3c49` a dark blue as the text that will be on them.
- Dimensions should be `1600 x 900`  `16:9`

#### Standard Fonts 
- [Comfortaa](https://fonts.google.com/specimen/Comfortaa?query=comf&selection.family=Comfortaa)
- [Pompiere](https://fonts.google.com/specimen/Pompiere?query=pomp&selection.family=Pompiere)
- [Nunito](https://fonts.google.com/specimen/Nunito?query=nun&selection.family=Nunito)
([link to all three fonts](https://fonts.google.com/specimen/Pompiere?query=pomp&selection.family=Comfortaa|Nunito|Pompiere))

#### Standard Colors
- `#0b3c49` a dark blue used for text
- `#f2a130` an orangey yellow
- `#444444` a dark gray
- `#86b500` greenish
- `#f4f4f4` almost white, kinda gray

### Proposed workflow:
1. Fork this repo
2. Add images to `img_bg` keeping to the existing naming conventions:
   * Each image should have the twitter handle and meet accessibility standards
   * follow the examples in `img_bg` 
3. Once you are done and checked for typos, commit it & push your work to your fork
4. Submit a PR against this repo

Someone will look it over and, if deemed appropriate, will merge your PR.

Thanks for contributing!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Comment by VickiLanger** [comment]
Started in: `generate_stripey_backgrounds.py`

https://github.com/VickiLanger/LGBTQ-of-the-day-bot/blob/production/generate_stripey_backgrounds.py

heavily influenced (mostly the same as) by https://alexwlchan.net/2020/03/stripey-flag-wallpapers/
